pandoc.cabal

```diff
@@ -598,7 +598,6 @@ library
                    Text.Pandoc.UTF8,
                    Text.Pandoc.Templates,
                    Text.Pandoc.XML,
-                   Text.Pandoc.Network.HTTP,
```

Comment:
Shouldn't this now appear in other-modules?

Reply:
Sorry about that; fixed.